notebooks/official/pipelines/google_cloud_pipeline_components_bqml_text.ipynb

```diff
@@ -0,0 +1,1460 @@
+{
```

Comment:
<p>Add the steps here, like:</p><p> </p><p>1) Get dataset from UCI archive.</p><p>2) Untar the dataset</p><p>3) Copy the dataset to a Cloud Storage location.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/403/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
done